Zend/zend_string.c

```diff
@@ -76,7 +76,7 @@ static void zend_init_interned_strings_ht(HashTable *interned_strings, int perma
 ZEND_API void zend_interned_strings_init(void)
 {
 	char s[2];
-	int i;
+    size_t i;
```

Comment:
This is really an `unsigned char`, but it's not easy to iterate over that. Rather than `size_t` I would use `unsigned` here, as this is unrelated to sizes or offsets.